src/Plugin/Condition/PathHasAlias.php

```diff
@@ -72,11 +73,18 @@ public static function create(ContainerInterface $container, array $configuratio
   }
 
   /**
-   * {@inheritdoc}
+   * Check if path has alias.
+   *
+   * @param string $path
+   *   The path to check.
+   * @param mixed $language
+   *   The language to check for a path in or NULL.
+   *
+   * @return bool
+   *   TRUE if the path exists in that language.
    */
-  public function evaluate() {
-    $path = $this->getContextValue('path');
-    $language = $this->getContext('language')->hasContextValue() ? $this->getContextValue('language')->getId() : NULL;
+  public function doEvaluate($path, LanguageInterface $language = NULL) {
+    $language = !is_null($language) ? $language->getId() : NULL;
```

Comment:
What should it be, then? Because just taking it out creates an error because getAliasByPath expects a string, not a Language object. And putting $language->getId() in the getAliasByPath call errors out when $language is NULL, obviously.